liquibase-service/src/main/resources/db/rdb_modern/tables/025-create_nrt_srte_totalidm-001.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE dbo.nrt_srte_TotalIDM (
```

Comment:
We can add the IF NOT EXISTS BEGIN END so that this code doesn't error out on any Liquibase rebuilds.

Reply:
@upasanapattnaik-eq Not all scripts need BEGIN END. If there are multiple/block of  statements that need to be run after table creation we would need that . According for few scripts I have added BEGIN END.